wp-content/themes/cjet/guides.php

```diff
@@ -54,25 +54,30 @@
 					<ul class="guide-tree">
 						<?php echo $children; ?>
 					</ul>
+					
+					<ul class="guide-sidebar-below-toc-widget-area">
+						<?php dynamic_sidebar( 'guide-sidebar-below-toc' ); ?>
+					</ul>
 
 					<?php
 					// on interior guide pages, list resources attached to the parent guide page
-					if ( $attachments ) : ?>
-					<div class="resources">
-						<h4><?php _e('Resources', 'cjet'); ?></h4>
+					// if ( $attachments ) : ?>
```

Comment:
If we're removing the iterator for attachments, then the query that populates $attachments should also be removed

Reply:
In slack:

> i just had it commented it out for now until we decide if attachments are going in the sidebar or under the content